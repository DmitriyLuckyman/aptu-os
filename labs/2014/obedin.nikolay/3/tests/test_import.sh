#!/usr/bin/env bash

source $(dirname $0)/common.sh

test() {
    run ./import $FS_DIR $FILES_DIR/file1 /file1
    rm -rf $FILES_DIR/non_existing_file
    run ./import $FS_DIR $FILES_DIR/non_existing_file /non_existing_file
}
