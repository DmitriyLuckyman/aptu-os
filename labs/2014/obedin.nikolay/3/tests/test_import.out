Error: can not open source file
